### PR TITLE
Make Key-Pair argument optional for GCP

### LINKED ISSUE #67: Make key-pair directory path argument optional to Lucy

### PR DESCRIPTION
**Pull request by ejMina226**
Explain your changes:
*

Explain how you tested your changes:
*

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #67

### CODE REVIEW

File: src/lib/integration_test_cloud_engine/mina_automation.ml [hunk 1]
```diff
@@ -158,7 +155,7 @@ module Network_config = struct
           mp
       | hd :: tl ->
           let ( { Test_config.Test_Account.balance; account_name; timing; _ }
```
Comment (ejMina226) [inline src/lib/integration_test_cloud_engine/mina_automation.ml:157]: I had to put in a dummy argument of _ to cover the pk as you've changed the test_config. Will GCP still work like this?

Comment (Isaac-DeFrain) [reply src/lib/integration_test_cloud_engine/mina_automation.ml:157]: I'm not sure what you're referring to... I don't see any additional dummy arg

**Review by Isaac-DeFrain — APPROVED** [review]
LGTM

File: src/lib/integration_test_cloud_engine/mina_automation.ml (hunk 1 above)
Comment (ejMina226) [reply src/lib/integration_test_cloud_engine/mina_automation.ml:157]: @Isaac-DeFrain I was referring to the underscore `_` after `timing`. It wasn't there before. For some reason git isn't picking up on the change.

**Pull request merged**